src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -66,14 +66,27 @@ def ffmpeg_read(bpayload: bytes, sampling_rate: int) -> np.array:
     return audio
 
 
-def apply_stride(tokens, stride):
-    max_token_n = tokens.shape[-1]
+def audio_to_logits(tokens_or_logits, stride):
```

Comment:
What exactly does the function do? Could we add some docstring? Also I don't understand the name `audio_to_logits` really

Reply:
It's just change the stride number to go from `audio space` (10s at 16_000  means (160_000, 8_000, 8_000)) for instance. to `logits_space` (2333, 160, 160) for instance.

Do you think of a better name ? Doctstring could help a little.

Reply:
Ah I see. Maybe `get_output_stride_from_input_stride(input_shape, stride)` and directly pass the shape? Yeah think a little docstring can help here